src/extension/extension.ts

```diff
@@ -205,7 +205,7 @@ export class RunmeExtension {
         const cell = context.globalState.get<number>(EXECUTION_CELL_STORAGE_KEY)
```

Comment:
It might be wise to not just add the cell but also a timestamp, potentially discard the operation if the entry is e.g. >5min old. Otherwise this could lead to weird side effects.